### PR TITLE
Add policy.cfg file for scheduler test

### PR DESCRIPTION
**Pull request by kasturinarra**

### CODE REVIEW

**Comment by zhouying7780** [comment]
LGTM

File: scheduler/policy.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+"kind" : "Policy",
+"apiVersion" : "v1",
+"predicates" : [
+],
+"priorities" : [ 
+]
+}
```
Comment (xingxingxia) [inline scheduler/policy.cfg]: Since it is json, better align well and use .json

Comment (kasturinarra) [reply scheduler/policy.cfg]: The scheduler configuration file is a JSON file, which must be named policy.cfg, that specifies the predicates and priorities the scheduler will consider.

Above is the reason for having it that way and not .json

Comment (zhouying7780) [reply scheduler/policy.cfg]: No, seems this if force-code , can't use .json.

Comment (zhouying7780) [reply scheduler/policy.cfg]: https://docs.openshift.com/container-platform/4.3/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-creating_nodes-scheduler-default

Comment (xingxingxia) [reply scheduler/policy.cfg]: Oh not so, oc create configmap -n openshift-config --from-file=**policy.cfg**=/path/to/file

Comment (kasturinarra) [reply scheduler/policy.cfg]: okay, if that is the case, then i think i will just send an update to this

**Pull request merged**